.github/workflows/bump_torch_mlir.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Automatically bumps the torch-mlir submodule to match shark-infra/torch-mlir:main.
```

Comment:
nit: I usually put a break between the license header comment block and file comments

```suggestion
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

# Automatically bumps the torch-mlir submodule to match shark-infra/torch-mlir:main.
```